hnn_core/feed.py

```diff
@@ -94,7 +94,7 @@ def set_prng(self, seed=None):
         if hasattr(self, 'seed2'):
             self.prng2 = np.random.RandomState(self.seed2)
 
-    def set_event_times(self, inc_evinput=0.0):
```

Comment:
also got rid of this as per our discussion @rythorpe 

it's not doing anything anyway